dockstore-webservice/src/main/resources/migrations.1.7.0.xml

```diff
@@ -162,10 +162,6 @@
     <changeSet author="natalieperez (generated)" id="changeLastBuildColumnName">
         <renameColumn tableName="tag" oldColumnName="lastmodified" newColumnName="lastbuilt"></renameColumn>
         <setColumnRemarks tableName="tag" columnName="lastbuilt" remarks="For automated builds: The last time the container backing this tool version was built. For hosted: N/A"></setColumnRemarks>
-        <!-- Set lastbuilt to null for versions/tags of hosted tools -->
```

Comment:
This isn't going to work on deploys to staging. 
changeLastBuildColumnName as a changeset won't be re-run since the database tracks and skips changesets that have already been run

Reply:
Hmm...it may look like dbUpdateDate/dbCreateDate is the only option then, since that information is lost forever.